<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="172.16.0.2"/>
  <arg name="load_gripper" default="true" />
  <!-- <node pkg="rosbag" type="record" name="bag_record" 
        args="/ARCmodel_reaching/debug_msg -o /home/fyw/Documents/projects/panda_ros/franka_panda_control_ws/data/armrc_out.bag"/> -->

  <include file="$(find ati_sensor)/launch/ft_sensor.launch" />

  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="ARCmodel_reaching"/>

  </node>
</launch>
